velox/common/caching/SsdFile.cpp

```diff
@@ -920,6 +924,7 @@ void SsdFile::readCheckpoint(std::ifstream& state) {
       entries_[std::move(key)] = run;
     }
   }
+  ++stats_.checkpointsRead;
```

Comment:
Put this at the function entry?

Reply:
Do we want this counter to include failed reads? I think before this point the read can still be interrupted with exception.